User-Help/GlobalTrustLevels.md

```diff
@@ -0,0 +1,138 @@
+| **Suspend specific user permissions**     | n/a                                                                                  | n/a                                                                                                  | n/a                                              | n/a                                                                                                 | n/a                                                                               | Full access                                                    |
+| **Sent private messages to users**        | n/a                                                                                  | n/a                                                                                                  | n/a                                              | n/a                                                                                                 | n/a                                                                               | Full access                                                    |
+| **Annotate users**                        | n/a                                                                                  | n/a                                                                                                  | n/a                                              | n/a                                                                                                 | n/a                                                                               | Full access                                                    |
+| **View private messages and annotations** | n/a                                                                                  | n/a                                                                                                  | n/a                                              | n/a                                                                                                 | n/a                                                                               | Full access                                                    |
+| **Access to all other moderation tools**  | n/a                                                                                  | n/a                                                                                                  | n/a                                              | n/a                                                                                                 | n/a                                                                               | Full access                                                    |
+
+
+
+# Glossary of terms
+
+### Prerequisites
+
+Minimum activity required to reach a given trust level.
+
+### Posts
+
+Typically posts are questions. Depending on the category, however, they can also include other formats like blogs or wikis.
```

Comment:
Answers are posts too.  Questions and answers are almost certainly going to be the large majority of posts on any site.

Reply:
Can you protect an answer w/o protecting the top-level post?

Reply:
Protection prevents answers, so it only applies to questions (like hold and dupe only apply to questions).  We haven't really worked through protection yet and it's not planned for MVP, so maybe we don't want to talk about protection yet?

Reply:
Sure, I'll remove it entirely for now.